modules/integration/fsx-lustre-on-eks/app.py

```diff
@@ -22,13 +22,15 @@ def _param(name: str) -> str:
 eks_admin_role_arn = os.getenv(_param("EKS_CLUSTER_ADMIN_ROLE_ARN"))
 eks_oidc_arn = os.getenv(_param("EKS_OIDC_ARN"))
 eks_cluster_sg_id = os.getenv(_param("EKS_CLUSTER_SECURITY_GROUP_ID"))
+eks_handler_role_arn = os.getenv(_param("EKS_HANDLER_ROLE_ARN"))
 
 fsx_file_system_id = os.getenv(_param("FSX_FILE_SYSTEM_ID"))
 fsx_security_group_id = os.getenv(_param("FSX_SECURITY_GROUP_ID"))
 fsx_mount_name = os.getenv(_param("FSX_MOUNT_NAME"))
 fsx_dns_name = os.getenv(_param("FSX_DNS_NAME"))
 
 fsx_storage_capacity = int(os.getenv(_param("FSX_STORAGE_CAPACITY"), 1200))
+dra_export_path = os.getenv(_param("DRA_EXPORT_PATH"))
```

Comment:
is only export path needed? no import?

Reply:
In my use-case/testing, since ray was writing back to FSX -> S3, i was using only `dra_export_path`. I haven't experimented the `dra_import_path`, but the code is there.